_posts/2018-08-06-Kickstarter-Trajectory-2018-Summer.md

```diff
@@ -0,0 +1,175 @@
+
+### Cloud
+The last technical session we followed was about different cloud technologies. This was given by [Dieter Hubeau](/author/dieter-hubau/) and [Bas Moorkens](/author/bas-moorkens/).
+We learned about the advantages of running applications in the cloud and what the differences are between the different operation models.
+To make this more tangible, we went into multiple cloud platforms to see what the possibilities were. 
+At the end we focussed on Openshift Origin as this is one of the preferred container management packages inside JWorks.
+
+## Soft Skills
+Besides sharpening our technical skills we worked on our soft skills as well.
+In the 2 day session 'Essentials of Communication' we learned how to present ourselves by means of role playing games and cases that reflect real world scenarios.
+After an additional 'Brand Yourself' session we were ready to prove and present ourselves as worthy consultants against the management of Ordina.
+All these techniques are also useful in the Agile & Scrum methodologies where we learned the importance of being prepared for change.
+
+
+## DevCase
+<img alt="Devcase" src="{{ '/img/2018-08-06-Kickstarter-Trajectory-2018-Summer/fe-preview1.png' | prepend: site.baseurl }}" class="image fit">
+
+####  Introduction
+In the second month of the kickstarter trajectory we were assigned to develop an event planner.
+The purpose was to have more of an overview and control of the upcoming JWorks events.
+In short, JWorks employees can create and approve events depending on their rights.
+In addition, a weekly digest of the events is sent to the Telegram JWorks chat group and a bot is made available with some defined commands.
+
+#### Technology & Methodologies
+Together with our coaches [Orjan De Smet](/author/orjan-de-smet){:target="_blank"}, [Axel Bergmans](/author/axel-bergmans){:target="_blank"} and Haroun Pacquée we started off with an introduction to the project where the user stories on the scrum board were presented.
+After defining the sprint goal for the coming two weeks we divided ourselves into a frontend and a backend group. 
+Using scrum methodology we held our daily stand up meetings and as soon as new functionality was developed a pull request was made and reviewed by our coaches.
+Every two weeks at the end of the sprint a demo was shown to our coaches followed by a retrospective and a sprint planning.
+By making use of continuous integration code changes in Github were automatically deployed in Openshift where a Jenskins pipeline checked the different stages from testing the code and the code quality to the build of the docker image.
+
+Frontend tools that were used in the project:
+ * Angular
+ * Angular-CLI
+ * Angular Material
+ * Jest 
+ 
+In the backend we made use of the following technologies:
+ * Java 8
+ * Spring Boot
+ * Mockito and JUnit
+ * Telegram API
+ * Keycloak Security
```

Comment:
I think it would be interesting to also talk a bit more about Keycloak and the security setup.